prisma/schema.prisma

```diff
@@ -418,6 +419,22 @@ model TagsNews {
   @@map("tags_news")
 }
 
+model Ping {
```

Comment:
I believe it should be a m-n relationship. Let imagine, user A can send a message to n users, user A also can received n messages from n users.